airbyte-integrations/connectors/source-close-com/pyproject.toml

```diff
@@ -3,7 +3,7 @@ requires = [ "poetry-core>=1.0.0",]
 build-backend = "poetry.core.masonry.api"
 
 [tool.poetry]
-version = "0.5.3"
+version = "0.5.4"
```

Comment:
@jtruty can you run `poetry lock` in this path to regenerate the dependency file? Also please update the `docs/integrations/source-close` changelog documentation.

Reply:
done!